azure-mediaservices-spring-boot-starter/README.md

```diff
@@ -22,6 +22,15 @@ azure.mediaservices.account-name=put-your-media-services-account-name-here
 azure.mediaservices.account-key=put-your-media-services-account-key-here
 ```
 
+# Optional
+If you are using network proxy then add below properties to run azure media services behind proxy
```

Comment:
Missing the period at the end of sentence. 

BTW, are you a user of the Media Services? If yes, what's the reason for using proxy in your case?